Ventcamp_HTML/assets/js/fetch.js

```diff
@@ -20,7 +20,8 @@ function fetchFromDeck(field, params, processDataFromDeck) {
 function processMember(member) {
   // Dom Load hack
   setTimeout(function() {
-    $("#team > div").append(html)
+    if( member.name != "ToolBot!" )
```

Comment:
When comparing strings in Javascript, try using `===` `!==` otherwise the interpreter will try to make type conversions.

Reply:
duly noted!